Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ however, it has to be formatted properly to pass verification tests.
 
 ## [Unreleased] - yyyy-mm-dd
 
-
+### Changed
+- Project-Wide Input Actions can no longer be disabled (removed the UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS define)
```

Comment:
I would change this statement to say it can no longer be excluded from compilation. It is very possible to disable it by not marking any .inputactions asset as project-wide.